.github/workflows/cypress.yml

```diff
@@ -36,78 +36,60 @@ jobs:
     environment: ${{ inputs.environment }}
     strategy:
       matrix:
-        browser: [
-          "edge"
-        ]
+        browser: ["edge"]
     container:
       image: cypress/browsers:22.12.0
-    defaults:
-      run:
-        working-directory: Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests
     steps:
       - uses: actions/checkout@v4
         with:
           ref: ${{ github.ref }}
 
-      - name: Prepare Cypress cache
-        uses: cypress-io/github-action@v6
-        with:
-          runTests: false
-          browser: ${{ matrix.browser }}
-          working-directory: ./Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests
-
-      - name: Run (dev)
-        if: inputs.environment == 'dev'
-        uses: cypress-io/github-action@v6
-        env:
-          CYPRESS_db: ${{ secrets.DB_CONNECTION_STRING }}
-          CYPRESS_url: ${{ secrets.AZURE_ENDPOINT }}
-          CYPRESS_cypressTestSecret: ${{ secrets.CYPRESS_TEST_SECRET }}
-          CYPRESS_academisationApiUrl: ${{secrets.CYPRESS_ACADEMISATION_API_URL}}
-          CYPRESS_academisationApiKey: ${{ secrets.CYPRESS_ACADEMISATION_API_KEY}}
-        with:
-          browser: ${{ matrix.browser }}
-          working-directory: ./Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests
-          wait-on: ${{ secrets.AZURE_ENDPOINT }}
-          install: false
+      
+    
 
-      - name: Run (staging)
-        if: inputs.environment == 'staging'
+      
```

Comment:
Too many extra spaces. Just one empty line after the previous step is sufficient :)